src/couch_log/src/couch_log.erl

```diff
@@ -82,6 +66,7 @@ set_level(Level) ->
 
 -spec log(atom(), string(), list()) -> ok.
 log(Level, Fmt, Args) ->
+    couch_stats:increment_counter([couch_log, level, Level]),
```

Comment:
Wrong place. You need to move it inside of `true` clause down here, to not count un-called log messages for severity below a threshold.